forge-gui/res/cardsfolder/upcoming/passageway_seer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Passageway Seer
+ManaCost:3 B
+Types:Creature Tiefling Warlock
+PT:2/2
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigInitiative | TriggerDescription$ When CARDNAME enters the battlefield, you take the initiative.
+SVar:TrigInitiative:DB$ TakeInitiative
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of your end step, if you have the initiative, put a +1/+1 counter on CARDNAME.
```

Comment:
Should probably be done as Trigger Condition.

Also needs extra check because it is a special If clause

Reply:
Good call – I'll fix the other ones, too